From 2be1100514765be1b79ae140379dbe4c7a6d0c94 Mon Sep 17 00:00:00 2001
From: Rob Clark <rclark@redhat.com>
Date: Wed, 30 Jul 2014 15:40:59 -0400
Subject: [fs] dcache: missing EXPORT_SYMBOL(simple_dname)

Message-id: <1406734861-25152-6-git-send-email-rclark@redhat.com>
Patchwork-id: 86840
O-Subject: [RHEL7 drm 5/7] fs: missing EXPORT_SYMBOL(simple_dname)
Bugzilla: 1119339
RH-Acked-by: Jerome Glisse <jglisse@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Dave Airlie <airlied@redhat.com>

RHBZ: 1119339

Note that upstream, this is part of commit:
31bbe16f6d88622d6731fa2cb4ab38d57d844ac

Which is a drm patch with this one line change in fs/dcache.c.  This
possibly could be squashed into the drm rebase patch when merging if
you prefer.

CC: Jarod Wilson <jarod@redhat.com>
CC: Jeff Layton <jlayton@redhat.com>
CC: Larry Woodman <lwoodman@redhat.com>
CC: Rafael Aquini <aquini@redhat.com>
Signed-off-by: Rob Clark <rclark@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/dcache.c b/fs/dcache.c
index 503c9dd..f2d03e0 100644
--- a/fs/dcache.c
+++ b/fs/dcache.c
@@ -2938,6 +2938,7 @@ char *simple_dname(struct dentry *dentry, char *buffer, int buflen)
   end = ERR_PTR(-ENAMETOOLONG);
  return end;
 }
+EXPORT_SYMBOL(simple_dname);
 
 /*
  * Write full pathname from the root of the filesystem into the buffer.
-- 
1.7.1